src/stores/AppStore/AppStore.ts

```diff
@@ -1904,7 +1911,7 @@ export class AppStore {
         // Set overlay defaults from current frame
         autorun(() => {
             if (this.activeFrame) {
-                this.overlayStore.setDefaultsFromAST(this.activeFrame);
+                this.overlayStore.setDefaultsFromFrame(this.activeFrame);
```

Comment:
> This PR also fixes a inconsistency between `defaultSystem`(overlayStore) and the defined system in the image header. This inconsistency happens after the user explicitly specified a coordinate system. Then if the user loads a new image or changes the active frame, the `defaultSystem`(overlayStore) will fixed to the specified system.

@kswang1029 After we fixed the bug, the coordinate system changes with the active image when it's set to AUTO, which is different from the dev branch. Not sure if this is the desired behavior.